calypso/service.go

```diff
@@ -103,41 +105,51 @@ type vData struct {
 	Signature *darc.Signature
 }
 
-// ProcessClientRequest implements onet.Service. We override the version
-// we normally get from embeddeding onet.ServiceProcessor in order to
-// hook it and get a look at the http.Request.
-func (s *Service) ProcessClientRequest(req *http.Request, path string, buf []byte) ([]byte, *onet.StreamingTunnel, error) {
-	if !disableLoopbackCheck && path == "Authorise" {
-		h, _, err := net.SplitHostPort(req.RemoteAddr)
-		if err != nil {
-			return nil, nil, err
-		}
-		ip := net.ParseIP(h)
-
-		if !ip.IsLoopback() {
-			return nil, nil, errors.New("authorise is only allowed on loopback")
-		}
-	}
-
-	return s.ServiceProcessor.ProcessClientRequest(req, path, buf)
+// Authorise is deprecated - please use Authorize.
```

Comment:
For the sake of strong backwards compatibility, can we keep this one working with localhost requests ? But still marked as deprecated.

Reply:
Done in latest commit.